news/2019-09-29-project-loved-week-of-september-29th.md

```diff
@@ -0,0 +1,165 @@
+
+[![](/wiki/shared/news/2019-09-29-project-loved-week-of-september-29th/mania/3-hype-lapix-remix.jpg)](https://osu.ppy.sh/community/forums/topics/966969)
+
+[PSYQUI feat. Such - hype (lapix Remix)](https://osu.ppy.sh/beatmapsets/871622#mania) by [Reba](https://osu.ppy.sh/users/3087654)  
+175 BPM, 4:37 | [7K] 7.05★
+
+Something worth hyping about is coming this round for 7K!
+
+Partly known for its appearance in the last 7K MWC Semifinals tiebreaker, Reba's hype is a fairly stamina-heavy chart utilizing long, hyper dense chordstreaming at 175 BPM. With a couple of minijacks and LNs sprinkled here and there as well as a forgiving rest section, the map leads up to an LN reliant chordstream section. Not really the kind of LNs you'd expect from an O2Jam chart but rather leaning more towards short LN chordstreams and small LN layers, fairly reminiscent of the meta with high difficulty mapping meta for osu!mania 7K.
+
+If dense chordstreaming is your thing, then you have to try this map out!
+
+---
+
+From everyone on the Project Loved team, we hope you enjoyed this week's selection, and are looking forward to the next! The winning beatmaps will hit the [Loved beatmap listing](https://osu.ppy.sh/beatmapsets?s=loved) sometime next week and will be announced via [@osugame on Twitter](https://twitter.com/osugame).
+
+In addition to all of the Loved captains that choose the maps and write the descriptions, make sure to thank [Toy](https://osu.ppy.sh/users/2757689), [Noffy](https://osu.ppy.sh/users/1541323), [huu](https://osu.ppy.sh/users/6044237) and [Ephemeral](https://osu.ppy.sh/users/102335) for their miscellaneous help in managing Project Loved.
+
+If you have any suggestions for beatmaps to be added to the Loved category, please submit them using the [Google form](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)! You can view the current submissions on the corresponding [Google sheet](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit#gid=0). For osu!mania, please use [this Google form](https://docs.google.com/forms/d/e/1FAIpQLSeaGfoQNGMqw4qQcqRPItUZILh2fGwJR6ly6cZNY9OWPXkFhw/viewform) and [this Google sheet](https://docs.google.com/spreadsheets/d/1sjkTwUSvQ5Me-6rK61rToTg2bU-yX9X29CXdzttvhtM/edit) instead.
+
+—the Project Loved team
```

Comment:
considering credit is given to everyone within the post already i think it'd be okay to change this to Toy (you already use "I" sometimes when writing these anyway)
```suggestion
—Toy
```
if u change this I'll change program too